source/fuel_client.lua

```diff
@@ -2,7 +2,8 @@ local isNearPump = false
 local isFueling = false
 local currentFuel = 0.0
 local currentCost = 0.0
-local currentCash = 1000
+local todaycoast = 0
```

Comment:
```suggestion
local todaycost = 0
```
Typo fix for clarity